app/views/partials/lib/commands/posts/update/build.liquid

```diff
@@ -8,7 +8,6 @@
     "upvote_profile_ids": {{ object.upvote_profile_ids | json }},
```

Comment:
how about those properties ? why do we set them here @piotrze ? I know it's not part of the PR, but it looks like we shouldn't be doing that ?

Reply:
I'm using this mutation for upvote/downvote. Is this causing problems?

Reply:
if you update the question, doesn't it clear the array? this comes from user input, correct?

Reply:
ah, it comes from graphql, so it's just an unnecessary assignment probably then. Should be fine. We had a problem with the score because graphql was not returning it.